kuksa-persistence-provider/Readme.md

```diff
@@ -83,20 +83,27 @@ As the statestore does not make a difference between current and target (actuati
 
     json_djson = [ "dep:djson","dep:tinyjson"  ]   # Uncommment to use djson
     ```
+
   * Install rust nightly which is needed for djson at the moment [11/2024]
-    ```bash
+
+  * ```bash
     rustup toolchain install nightly
     rustup default nightly
     ```
-  * Enable with ```cargo build --features json_djson --no-default-features``````
 
+  * Enable with ```cargo build --features json_djson --no-default-features``````
```

Comment:
Remove the last 3 characters
![image](https://github.com/user-attachments/assets/9bb6e453-ff57-441f-b8d5-47106591e7d2)

Reply:
Done